### DISCUSSION TITLE
第5节课作业(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#305] [作业]
Here

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135591121

**Comment by ltstar** [comment]
[Uploading 课程5-LMDeploy.md…]()

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/33995527b86646edb9c5108b69d0fd2f

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135660122

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17974745

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec5/

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135758850?spm=1001.2014.3001.5501

**Comment by sunhanyu714** [comment]
https://github.com/sunhanyu714/Internlm_learning/blob/main/%E7%AC%AC%E4%BA%94%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A

**Comment by 0-yy-0** [comment]
https://blog.csdn.net/qq_34354951/article/details/135834112

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/A6jqdySqAo33BBxCfMacDF3snOM
